tests/bdd/conftest.py

```diff
@@ -76,8 +76,8 @@ def driver():
 
 
 # Close firefox after all tests are completed
-def pytest_sessionfinish(session, exitstatus):
-    web_driver.quit()
+# def pytest_sessionfinish(session, exitstatus):
```

Comment:
Please uncomment lines 79 and 80 to ensure that firefox gets closed after tests are done on bob.

Reply:
good catch, I forgot to do that.  Just pushed a new commit to uncomment those.

Reply:
wait, sorry this is a different file. I'll fix that in a second